gtsofa/medium-api
# #166213689 Integrate TravisCI and update ReadMe

### What is this PR supposed to do?
- Integrate TravisCI with ReadMe badge

### How should this be manually tested?
- clone the repo and navigate to the project directory
- check out to  this `ch-set-up-app-test-166213696` branch
- check the readme for the badge

### Relevant PT stories
- [166213689](https://www.pivotaltracker.com/n/projects/2350589/stories/#166213689)
